content/docs/command-reference/plots/diff.md

```diff
@@ -21,7 +21,8 @@ positional arguments:
 
 This command is a way to visualize the "difference" between metrics among
 experiments in the <abbr>repository</abbr> history, by plotting multiple
-versions of the metrics. All plots defined in `dvc.yaml` are used by default.
+versions of the metrics. All plots defined in `dvc.yaml` are used by default. If
+`targets` are specified, they do not have to be defined in `dvc.yaml`.
```

Comment:
> If `targets` are specified, they do not have to be defined in `dvc.yaml`.

Hmmm... There's no `targets` argument in this command. We went full circle here I think: a similar note was added initially, then we said it should be in the `--targets` option description, and then decided it shouldn't be needed there either as long as it was clear that all plots commands work with any valid plots file (whether in dvc.yaml or not).

Reply:
We went full circle here I think: a similar note was added initially, then we said it should be in the `--targets` option description, and then decided it shouldn't be needed there either as long as it was clear that all plots commands work with any valid plots file (whether in dvc.yaml or not). Now we're back with a note in the description.

Reply:
I would move this note to before "The plot style can be customized" for now (and use the phrase "`--targets` option" to be clear).

Reply:
OK, I took care of this in dca051e8.